config/locales/en.yml

```diff
@@ -52,3 +56,12 @@ en:
     search:
       fields:
         cho_title_ssim: Title
+    resources:
+      upload:
+        form:
+          cho_title_ssim: Title
```

Comment:
Are you sure we need this line?  was this the result of a rebase?

Reply:
Yes, from rebase. Which line #s do you think can go?

Reply:
61 & 62 for certain.

Reply:
OK. Removed.